backend/src/services/secret-sharing/secret-sharing-service.ts

```diff
@@ -62,6 +63,7 @@ export const secretSharingServiceFactory = ({
 
     const newSharedSecret = await secretSharingDAL.create({
       name,
+      password,
```

Comment:
I don't think this is a good idea to keep this in plain text

Reply:
yeah, currently hashing it and adding a validate endpoint on the backend.